### PR TITLE
[ES]Corrections, AoA translate and Iceman translate

### PR DESCRIPTION
**Pull request by Freakmod**

### CODE REVIEW

**Comment by Freakmod** [comment]
i dont know why fail in Mutant genensis if only change the text to Spanish

File: translations/es/pack/mut_gen_encounter.json [hunk 1]
```diff
@@ -676,120 +622,121 @@
...
 	{
 		"code": "32149",
 		"name": "Burbuja Magnética",
-		"text": "Vincula esta carta a Magneto.\nMagneto gana Represalia 1.",
+		"text": "Vincula esta carta a Magneto.\nMagneto gana Represalia 1.\n<b>Interrupción obligada:</b> cuando Magneto vaya a sufrir cualquier cantidad de Daño, en vez de eso coloca ese Daño sobre esta carta. Luego, si hay al menos 8 de Daño aquí, descarta esta carta.",
 		"traits": "Magnético."
 	},
 	{
 		"code": "32150",
+		"flavor": "Vaya, esto no es bueno —Arcángel"
```
Comment (jordanweiler) [inline translations/es/pack/mut_gen_encounter.json:648]: Failure says
```
/home/runner/work/marvelsdb-json-data/marvelsdb-json-data/translations/es/pack/mut_gen_encounter.json: File is not valid JSON.
Expecting ',' delimiter: line 649 column 3 (char 39424)
```

It looks like a comma is missing at the end of this line.

Comment (Freakmod) [reply translations/es/pack/mut_gen_encounter.json]: in my file have the comma and the las t update is update that coma xD

Comment (Freakmod) [reply translations/es/pack/mut_gen_encounter.json]: correct now again and working...okey xD

**Comment by Freakmod** [comment]
the last update x.json are correct Traits in spanish

**Comment by Freakmod** [comment]
@zzorba any problem with my commit?

**Pull request closed without merging**